waku/node/v2/wakunode2.nim

```diff
@@ -265,11 +263,24 @@ proc setFilterPeer*(n: WakuNode, address: string) =
 
   n.wakuFilter.setPeer(remotePeer)
 
-proc connectToNodes*(n: WakuNode, nodes: openArray[string]) =
+proc connectToNodes*(n: WakuNode, nodes: seq[string]) {.async.} =
```

Comment:
What's the difference between when these two functions are used?

Reply:
It looks like one is calling on relay protocol specifically, maybe worth capturing that in the function name. Also not clear to me why there is a difference with PeerInfo and string here

Reply:
Both are calling dial with the relay protocol, I made a second function called connectToNodes for when we use it from tests or when we already have PeerInfo vs just the stringified version of the peer address. Otherwise we would need to stringify all before passing it or parsing all before passing.